src/pumps/MultiFlowPump.sol

```diff
@@ -249,11 +251,18 @@ contract MultiFlowPump is IPump, IMultiFlowPumpErrors, IInstantaneousPump, ICumu
         }
         bytes16[] memory lastEmaReserves = slot.readBytes16(numberOfReserves);
         uint256 deltaTimestamp = _getDeltaTimestamp(lastTimestamp);
-        bytes16 blocksPassed = (deltaTimestamp / BLOCK_TIME).fromUInt();
-        bytes16 alphaN = ALPHA.powu(deltaTimestamp);
         emaReserves = new uint256[](numberOfReserves);
+        // If no time has passed, return last EMA reserves.
+        if (deltaTimestamp == 0) {
+            for (uint256 i; i < numberOfReserves; ++i) {
+                emaReserves[i] = lastEmaReserves[i].pow_2ToUInt();
+            }
+            return emaReserves;
+        }
+        bytes16 capExponent = ((deltaTimestamp - 1) / CAP_INTERVAL + 1).fromUInt();
```

Comment:
`capExponent` should never = 0 unless `deltaTimestamp` is 0. If `CAP_INTERVAL` is set to 12 seconds to match the block time on Ethereum and then the Ethereum block time changes to say 10 seconds, we want to make sure that `capExponent` is still = 1 after block passes otherwise the oracle could be manipulated by forcing a Pump update every block. By rounding up, we are able to guarantee such a manipulation is not possible.